README.md

```diff
@@ -24,7 +26,63 @@ npm run lint
 ## Local development
 
 1. Create a new repository based on the `aem-boilerplate` template and add a mountpoint in the `fstab.yaml`
-1. Add the [AEM Code Sync GitHub App](https://github.com/apps/aem-code-sync) to the repository
-1. Install the [AEM CLI](https://github.com/adobe/aem-cli): `npm install -g @adobe/aem-cli`
-1. Start AEM Proxy: `aem up` (opens your browser at `http://localhost:3000`)
-1. Open the `{repo}` directory in your favorite IDE and start coding :)
+2. Add the [AEM Code Sync GitHub App](https://github.com/apps/aem-code-sync) to the repository
+3. Install the [AEM CLI](https://github.com/adobe/aem-cli): `npm install -g @adobe/aem-cli`
+4. Start AEM Proxy: `aem up` (opens your browser at `http://localhost:3000`)
+5. Open the `{repo}` directory in your favorite IDE and start coding :)
+
+## UDEX webcomponent integration
```

Comment:
Nit: This may be a one time thing, but we can call out that one needs to run `npm i` atleast once to get the necessary modules (like `vite`) installed.

Reply:
Add a comment saying one needs to run `npm install`